bedrock/press/templates/press/press-inquiry.html

```diff
@@ -1,104 +1,87 @@
-              <p class="error-msg">{{_('Please enter your name.')}}</p>
-            {% endif %}
-            {{ form.name }}
-          </div>
-
-          <div class="field required">
-            <label for="id_user_email">{{ _('Email Address') }}</label>
-            {% if form.user_email.errors %}
-              <p class="error-msg">{{_('Please enter a valid email address.')}}</p>
-            {% endif %}
-            {{ form.user_email }}
-          </div>
-
-          <div class="field required">
-            <label for="id_media_org">{{ _('Media Organization') }}</label>
-            {% if form.media_org.errors %}
-              <p class="error-msg">{{_('Please enter the name of your media organization.')}}</p>
-            {% endif %}
-            {{ form.media_org }}
-          </div>
-
-          <div class="field required">
-            <label for="id_inquiry">{{ _('Inquiry') }}</label>
-            {% if form.inquiry.errors %}
-              <p class="error-msg">{{_('Please enter the subject of your inquiry.')}}</p>
-            {% endif %}
-            {{ form.inquiry }}
-          </div>
-
-          <div class="field required">
-            <label for="id_deadline">{{ _('Deadline') }}</label>
-            {% if form.deadline.errors %}
-              <p class="error-msg">{{_('Please enter your deadline.')}}</p>
-            {% endif %}
-            {{ form.deadline }}
-            <div class="field-note">{{ _('2015-03-14 or March 14, 2015') }}</div>
-          </div>
+          <label for="id_jobtitle">{{ _('Job Title') }}</label>
+          {% if form.jobtitle.errors %}
+          <p class="error-msg">{{_('Please enter your job title.')}}</p>
+          {% endif %}
+          {{ form.jobtitle }}
+
+          <label for="id_name">{{ _('Name') }}</label>
+          {% if form.name.errors %}
+          <p class="error-msg">{{_('Please enter your name.')}}</p>
+          {% endif %}
+          {{ form.name }}
+
+          <label for="id_user_email">{{ _('Email Address') }}</label>
+          {% if form.user_email.errors %}
+          <p class="error-msg">{{_('Please enter a valid email address.')}}</p>
+          {% endif %}
+          {{ form.user_email }}
+
+          <label for="id_media_org">{{ _('Media Organization') }}</label>
+          {% if form.media_org.errors %}
+          <p class="error-msg">{{_('Please enter the name of your media organization.')}}</p>
+          {% endif %}
+          {{ form.media_org }}
+
+          <label for="id_inquiry">{{ _('Inquiry') }}</label>
+          {% if form.inquiry.errors %}
+          <p class="error-msg">{{_('Please enter the subject of your inquiry.')}}</p>
+          {% endif %}
+          {{ form.inquiry }}
+
+          <label for="id_deadline">{{ _('Deadline') }}</label>
+          {% if form.deadline.errors %}
+          <p class="error-msg">{{_('Please enter your deadline.')}}</p>
+          {% endif %}
+          {{ form.deadline }}
 
           {{ form.office_fax }}
```

Comment:
I believe this is a honey pot field, aimed at stopping spam. Should it be hidden visually?